recipes/simple-websocket-server/config.yml

```
@@ -0,0 +1,4 @@
+---
+versions:
+  "v2.0.0":
```

```suggestion
  "2.0.0":
```

Please, discard the prefix